src/components/PersonaCard/Jquery.PersonaCard.js

```diff
@@ -24,7 +24,7 @@
         $(this).addClass('is-active');
 
         /** Function for switching selected item into view by adding a class to ul. */
-        var updateForItem = function(item){
+        var updateForItem = function(wrapper, item){
           var wrapper = document.getElementById("detailList");
```

Comment:
OK.  Sorry about that.  I am trying to figure out how to make the new commit in SourceTree.

From: Eric Thompson [mailto:notifications@github.com]
Sent: Friday, December 18, 2015 4:36 PM
To: OfficeDev/Office-UI-Fabric <Office-UI-Fabric@noreply.github.com>
Cc: Todd Baginski <todd@toddbaginski.com>
Subject: Re: [Office-UI-Fabric] Office UI Fabric Issue 236 (#237)


In src/components/PersonaCard/Jquery.PersonaCard.js<https://github.com/OfficeDev/Office-UI-Fabric/pull/237#discussion_r48072642>:

> @@ -24,7 +24,7 @@

>          $(this).addClass('is-active');

>

>          /** Function for switching selected item into view by adding a class to ul. */

> -        var updateForItem = function(item){

> +        var updateForItem = function(wrapper, item){

>            var wrapper = document.getElementById("detailList");

It looks like you forgot to remove this line.

—
Reply to this email directly or view it on GitHub<https://github.com/OfficeDev/Office-UI-Fabric/pull/237/files#r48072642>.